### ISSUE TITLE
[DEV] Further issues with WEBM gallery gestures

### ISSUE THREAD

**Issue by Astridchan** [Adamantcheese/Kuroba#624]
After the fix to #557 there's a still a couple of inconsistencies with WEBM gallery gestures on the latest Dev build.

>The following will now be the functionality: Gifs are double tap to pause again, single tap to close. Players now don't display their controls by default, and a single tap will bring them up, and another tap will close the controls and resume playback. Double tapping a video will play/pause it. Also, videos will display if they are buffering.

To bring up and then hide the player controls it currently takes three taps:
1. tap brings player controls up
2. tap pauses playback
3. tap unpauses and hides controls

The second tap feels completely unnecessary, expected behaviour would be:
1. tap brings up controls
2. tap hides controls

Toggling player control visibility shouldn't affect playback.

Another way to pause playback with only a single tap:
1. Tap to bring up player controls
2. Wait until player controls hide automatically, then tap again
3. Player controls appear AND playback pauses

It feels like the app is counting successive taps as double taps even when there is several seconds in between.

Thirdly, double tapping pauses playback but it only takes a single tap to resume, for consistency unpause probably also should be a double tap.

So in summary, a single tap should only toggle the player control visibility and never affect playback. Pause and unpause is always a double tap, never a single tap.
